### PR TITLE
Added OCaml Support

### PR DESCRIPTION
**Pull request by adityaxdiwakar**
Added support for the OCaml language, image can be found here: https://andrewshitov.com/wp-content/uploads/2019/12/OCaml.png

### CODE REVIEW

File: python/vimsence.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 has_thumbnail = {
     'c', 'cr', 'hs', 'json', 'nim', 'ruby', 'cpp', 'go', 'javascript', 'markdown',
     'typescript', 'python', 'vim', 'rust', 'css', 'html', 'vue', 'paco', 'tex', 'sh',
-    'elixir', 'cs'
+    'elixir', 'cs', 'ocaml
```
Comment (anned20) [inline python/vimsence.py:39]: You forgot a closing quote.

```suggestion
    'elixir', 'cs', 'ocaml'
```

**Comment by hugolgst** [comment]
@adityaxdiwakar could you provide an image with a minimum size of 512x512 ?

**Comment by hugolgst** [comment]
Mention me when you can provide one

**Comment by anned20** [comment]
@hugolgst found one: https://github.com/ocaml/ocaml-logo/blob/master/Colour/PNG/colour-icon.png

**Pull request merged**